data/map planets.txt

```diff
@@ -4783,6 +4819,9 @@ planet "Varu K'est"
 	outfitter "Wanderer Advanced"
 	"required reputation" 15
 	bribe 0
+	tribute 1600
+		threshold 2000
```

Comment:
```suggestion
		threshold 5500
```
Seems odd to have such a low threshold when the tribute is quite high and the fleet is very large.